scripts/npm-cli-repos.txt

```diff
@@ -54,15 +47,15 @@ promzard
 query
 read
 read-cmd-shim
-read-package-json
 read-package-json-fast
+redact
 rfcs
 run-script
 ssri
 stafftools
 statusboard
-tap-nock
 template-oss
+types
```

Comment:
I think the `types` repo may have been manually excluded last time, unless it was recently tagged `npm-cli`. But leaving it in doesn't change the dependency graph.